packages/mobx-state-tree/src/core/type/type-checker.ts

```diff
@@ -116,9 +124,11 @@ export function typecheckPublic(type: IType<any, any>, value: any): void {
     const errors = type.validate(value, [{ path: "", type }])
 
     if (errors.length > 0) {
+        console.error("Failed to create `${type.name}` from:", value)
         fail(
-            `Error while converting ${prettyPrintValue(value)} to \`${type.name}\`:\n` +
-                errors.map(toErrorString).join("\n")
+            `Error while converting ${shortenPrintValue(
+                prettyPrintValue(value)
+            )} to \`${type.name}\`:\n\n` + errors.map(toErrorString).join("\n")
```

Comment:
sure about these double linebreaks? It might confusingly separate the error from the stack?

Reply:
not a perfect solution but it would make `at path /` really stand out in the scenario https://github.com/mobxjs/mobx-state-tree/issues/560 . I'm I sure that it would help.

Reply:
Ah got it. Maybe we should prefix each line with `\t` or `\t\t` as well to make it more stand out?

Reply:
I think an extra line break is enough in my case. `\t` is not necessary, although it might make it look better in some cases.

Reply:
would you mind adding the `\t` as well? Then we improve in as many cases as easily feasible

Reply:
I'm considering that using 4 spaces is better than using a `\t`?